Formula/ntjoin.rb

```diff
@@ -24,14 +22,13 @@ class Ntjoin < Formula
   uses_from_macos "libxml2"
   uses_from_macos "zlib"
 
-  fails_with :clang # needs openmp
-
   def install
     system "make", "-C", "src"
     ENV.prepend_path "PATH", libexec/"bin"
     xy = Language::Python.major_minor_version "python3"
     ENV.prepend_path "PYTHONPATH", libexec/"lib/python#{xy}/site-packages"
     inreplace "requirements.txt", "python-igraph", "python-igraph==0.7.1.post6"
+    inreplace "requirements.txt", "pybedtools", "pysam==0.15.4\npybedtools"
```

Comment:
Is this hack needed even when using clang on macOS rather than `gcc`?

Reply:
Yeah good point -- It installs successfully for me on my local mac without that line (after removing the `openmp` things), so removed it in my latest commit